app/api/helpers/pentabarfxml.py

```diff
@@ -44,39 +44,41 @@ def export(event_id):
                                      .filter_by(state='accepted')
                                      .filter(Session.deleted_at.is_(None))
                                      .order_by(asc(Session.microlocation_id)).distinct())
+
             for microlocation_id in microlocation_ids:
```

Comment:
May be you can rename this: ```microlocation_id``` to something like ```microlocation_array``` or `microlocation_tuple` etc depending on what it is. 
naming it `microlocation_id` and then reassigning it to `microlocation_id[0]` doesn't look good in terms of readability.